specs/phase0/p2p-interface.md

```diff
@@ -246,9 +246,8 @@ Likewise, clients MUST NOT emit or propagate messages larger than this limit.
 The `message-id` of a gossipsub message MUST be:
 
 ```python
-   message-id: base64(SHA256(message.data))
+   message-id: SHA256(message.data).
```

Comment:
Seems like the "." at the end was a typo

```suggestion
   message-id: SHA256(message.data)
```

Reply:
I am used to finish a sentence (even after a colon) with a dot within the math environment if the sentence is not continued afterwards, but we can remove that of course if you prefer it.